### PR TITLE
feat: update u-boot to 2021.01

### PR DESCRIPTION
**Pull request by guilhem**
Fix https://github.com/talos-systems/talos/issues/3186

### CODE REVIEW

**Comment by andrewrynhard** [comment]
Thanks @guilhem !

**Comment by smira** [comment]
Looks like build is broken on something deep in u-boot:

```
#81 31.40   BINMAN  all
#81 31.61 Traceback (most recent call last):
#81 31.61   File "/tmp/build/0/libretech_all_h3_cc_h5_u-boot/./tools/binman/binman", line 39, in <module>
#81 31.61     from binman import control
#81 31.61   File "/tmp/build/0/libretech_all_h3_cc_h5_u-boot/tools/binman/../binman/control.py", line 11, in <module>
#81 31.61     import pkg_resources
#81 31.61 ModuleNotFoundError: No module named 'pkg_resources'
#81 31.64 make: *** [Makefile:1024: all] Error 1
#81 ERROR: executor failed running [/toolchain/bin/bash -c set -eou pipefail
export BL31=$(pwd)/${SUN50I_A64_ARM_TRUSTED_FIRMWARE}/build/sun50i_a64/release/bl31.bin
cd ${LIBRETECH_ALL_H3_CC_H5_U_BOOT}
```

**Comment by smira** [comment]
I have a fix for this problem, it will take a bit to get all the pieces together

**Comment by guilhem** [comment]
Thank you @smira 
You are awesome

File: base/pkg.yaml [hunk 1]
```diff
@@ -10,6 +10,7 @@ steps:
       - |
         cp -R /toolchain/lib/gcc /lib
         cp -R /toolchain/lib/libgcc* /lib
+        cp -R /toolchain/lib/libz* /lib
```
Comment (smira) [inline base/pkg.yaml:13]: required for Python zlib extension to find `libz.so.1`

File: Pkgfile [hunk 2]
```diff
@@ -3,7 +3,7 @@
 format: v1alpha2
 
 vars:
-  TOOLS_IMAGE: ghcr.io/talos-systems/tools:v0.3.0-21-g0026740
+  TOOLS_IMAGE: ghcr.io/talos-systems/tools:v0.3.0-22-g4b418f3
```
Comment (smira) [inline Pkgfile:6]: updated Python image with pip bootstrapped (`setuptools`)

**Review by andrewrynhard — APPROVED** [review]

**Pull request merged**